samples/arduino/starterkit/Spaceship2.js

```diff
@@ -17,18 +17,20 @@
 var gpio = require('gpio');
 var pins = require('arduino101_pins');
 
+print("Starting Spaceship2.js");
+
 // set up the GPIO pins
 var button = gpio.open({
-    pin: pins.IO2,
+    pin: pins.IO4,
     direction: 'in',
     edge: 'any'
 });
 var led1 = gpio.open({
-    pin: pins.IO4,
+    pin: pins.LED0,
     direction: 'out'
 });
 var led2 = gpio.open({
-    pin: pins.IO7,
+    pin: pins.LED1,
```

Comment:
You probably didn't mean to include these changes, since it was just the way you were testing locally? But these are following the Arduino sample code which uses external LEDs so the pins used should not be changed.